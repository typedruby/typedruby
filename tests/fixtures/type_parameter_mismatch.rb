module Enumerable::[Mismatched]
end

class Array::[Nope]
end
