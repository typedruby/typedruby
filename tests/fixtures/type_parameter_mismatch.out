error: Mismatched type parameters

        @ __ROOT__/tests/fixtures/type_parameter_mismatch.rb:1
      1 |  module Enumerable::[Mismatched]
                  ^^^^^^^^^^^^^^^^^^^^^^^^ here
        @ (builtin stdlib):###
    ### |  module Enumerable::[EnumType]
                  ^^^^^^^^^^^^^^^^^^^^^^ previous definition here

error: Mismatched type parameters

        @ __ROOT__/tests/fixtures/type_parameter_mismatch.rb:4
      4 |  class Array::[Nope]
                 ^^^^^^^^^^^^^ here
        @ (builtin stdlib):###
    ### |  class Array::[ElementType] < Object
                 ^^^^^^^^^^^^^^^^^^^^ previous definition here
